salesagility/SuiteCRM
# Fixed #3655 - fix for popup compose view return message popup

## Description




fix for popup compose view return message popup

## Motivation and Context


see issue #3655 - SuiteCRM 7.9.0 - Sending email from Activity Subpanel error
also related to issue #3781 - Issue with return after sending an email

## How To Test This


see issue #3655:
Steps to Reproduce

Proceed to Case module
Click Compose Email under Activities subpanel
Input detail information and click on send/save as draft

## Types of changes


### Final checklist
